pptx_parser.py

```diff
@@ -0,0 +1,29 @@
+from pptx import Presentation
+
+
+class PptxParser:
+    """
+    Parsing a pptx file and yield the slide's text and index number.
+    """
+    @classmethod
+    def parse(cls, filepath):
```

Comment:
What about inserting the parsed text into a format string that will make it much more comprehensible?
did you compare different prompts and their results?